externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/SparkTBinaryFrontendService.scala

```diff
@@ -65,10 +63,8 @@ class SparkTBinaryFrontendService(
       respConfiguration.put("kyuubi.engine.id", sc.applicationId)
 
       if (req.getConfiguration != null) {
-        req.getConfiguration.asScala.get(KyuubiReservedKeys.KYUUBI_ENGINE_CREDENTIALS_KEY).filter(
-          _.nonEmpty).foreach { credentials =>
-          renewDelegationToken(credentials)
-        }
+        val credentials = req.getConfiguration.remove(KYUUBI_ENGINE_CREDENTIALS_KEY)
```

Comment:
Not all Map support `remove` operation, but here, it seems that the thrift request supports `remove`.